Makefile

```diff
@@ -91,7 +91,42 @@ build: build-server build-nightly-server build-ad-server build-client \
 .PHONY: build
 
 
-
+.PHONY: debug-vars
+debug-vars:
```

Comment:
IMHO `print-vars` or `report-vars` if a better target name.

Reply:
@synarete wrote:

> IMHO `print-vars` or `report-vars` if a better target name.

good idea! are you ok with changing this later in a follow-up PR?